NurVo_Frontend/src/pages/Home.tsx

```diff
@@ -114,33 +192,116 @@ function UserInfoHeader() {
   );
 }
 
+interface Review {
+  chapter_id: number;
+  chapter_name: string;
+  date: string;
+  step: number;
+  topic_name: string;
+}
+
+interface Todays {
+  chapter_id: number;
+  chapter_name: string;
+  date: string;
+  step: number;
+  topic_id: number;
+  topic_name: string;
+}
+
+
+export default function Home({ navigation, route }: HomeScreenProps) {
 
+  const [todays, setTodays] = useState<Chapter[]>([]);
+  const [reviews, setReviews] = useState<Chapter[]>([]);
+  const [numOfReview, setNumOfReview] = useState<number>(0);
 
-export default function Home({ navigation }: { navigation: any }) {
+  useEffect(() => {
+    stopSpeech();
+  }, [play]);
+
+  //todays lesson
+  useFocusEffect(
+    useCallback(() => {
+      const getData = async () => {
+        try {
+          const data = await fetchTodaysLesson();
+          if (!data) return;
+          const sectionData: Chapter[] = data.map((item: Todays) => ({
+            id: item.chapter_id,
+            name: item.chapter_name,
+            description: item.topic_name,
+            step: item.step
+          }));
+          setTodays([]);
+          setTodays(sectionData);
+  
+        } catch (error) {
+          console.log(error);
+        }
+      }
+      getData();
+    }, [])
+  );
+  //fetch reviews
+  useEffect(() => {
```

Comment:
useEffect(() => {
```suggestion
  useFocusEffect(
```
로 수정해야 학습 완료 후 홈화면으로 왔을 때 바로 리뷰가 업데이트 될 거 같습니다!!

Reply:
반영했습니다 감사합니다